.github/workflows/lint-and-test.yml

```diff
@@ -44,21 +44,33 @@ jobs:
         run: ./tools/run-isort-check
 
   pytest:
-    runs-on: ubuntu-latest
     strategy:
       # Not failing fast allows all matrix jobs to try & finish even if one fails early
       fail-fast: false
       matrix:
-        test-python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
-    name: Install & pytest - CPython ${{ matrix.test-python-version }}
+        env:
+          - {PYTHON: 3.5, OS: ubuntu-latest, NAME: "CPython 3.5 (ubuntu)"}
```

Comment:
Why is 3.5 kept though?

Reply:
There's a separate PR for migrating away from 3.5.